lib/domgen/sync/templates/abstract_sync_temp_population_impl.java.erb

```diff
@@ -164,29 +169,31 @@ public abstract class <%= data_module.sync.abstract_sync_temp_population_impl_na
+          "<%=j_escape_string(" LEFT JOIN #{ a.referenced_entity.sync.sync_temp_entity.sql.qualified_table_name } AS R ON R.#{ entity.sync.master_entity.attribute_by_name(:MappingID).sql.quoted_column_name } = S.#{ entity.sync.sync_temp_entity.attribute_by_name("#{a.name}MappingID").sql.quoted_column_name } AND R.#{ entity.sync.sync_temp_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } = S.#{entity.sync.sync_temp_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } ")%>" +
 <% elsif a.referenced_entity.abstract? -%>
 <%
   refs = []
   a.referenced_entity.concrete_subtypes.each_with_index do |subtype, index|
     refs << "R#{index}.ID"
 -%>
-          " LEFT JOIN <%= subtype.sync.master_entity.sql.qualified_table_name %> AS R<%= index %> ON R<%= index %>.MappingID = S.<%= entity.sync.sync_temp_entity.attribute_by_name("#{a.name}MappingID").sql.quoted_column_name %> AND R<%= index %>.MappingSource = S.MappingSource AND R<%= index %>.DeletedAt IS NULL " +
+           "<%=j_escape_string(" LEFT JOIN #{ a.referenced_entity.sync.master_entity.sql.qualified_table_name } AS R#{index} ON R#{index}.#{ entity.sync.master_entity.attribute_by_name(:MappingID).sql.quoted_column_name } = S.#{ entity.sync.sync_temp_entity.attribute_by_name("#{a.name}MappingID").sql.quoted_column_name } AND R#{index}.#{ entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } = S.#{entity.sync.sync_temp_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } AND R#{index}.#{ entity.sync.master_entity.attribute_by_name(:DeletedAt).sql.quoted_column_name } IS NULL ")%>" +
 <% end
   ref_id = "COALESCE(#{refs.join(',')})"
 -%>
 
 <% else -%>
-          "LEFT JOIN <%= a.referenced_entity.sync.master_entity.sql.qualified_table_name %> AS R ON R.MappingID = S.<%= entity.sync.sync_temp_entity.attribute_by_name("#{a.name}MappingID").sql.quoted_column_name %> AND R.MappingSource = S.MappingSource AND R.DeletedAt IS NULL " +
+           "<%=j_escape_string(" LEFT JOIN #{ a.referenced_entity.sync.master_entity.sql.qualified_table_name } AS R ON R.#{ entity.sync.master_entity.attribute_by_name(:MappingID).sql.quoted_column_name } = S.#{ entity.sync.sync_temp_entity.attribute_by_name("#{a.name}MappingID").sql.quoted_column_name } AND R.#{ entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } = S.#{entity.sync.sync_temp_entity.attribute_by_name(:MappingSource).sql.quoted_column_name } AND R.#{ entity.sync.master_entity.attribute_by_name(:DeletedAt).sql.quoted_column_name } IS NULL ")%>" +
```

Comment:
Pete, this line is causing issues in planner, but it doesn't seem to relate to my changes.
sql.table_name invoked on abstract entity Master.ResourceType
Error generating sync_master_ejb_impl:abstract_sync_temp_population_impl.java for data_module Master
RuntimeError
/Users/Ian/Code/planner/vendor/plugins/domgen/lib/domgen/core.rb:26:in `error'
/Users/Ian/Code/planner/vendor/plugins/domgen/lib/domgen/sql/model.rb:507:in `table_name'
/Users/Ian/Code/planner/vendor/plugins/domgen/lib/domgen/sql/model.rb:512:in `quoted_table_name'
/Users/Ian/Code/planner/vendor/plugins/domgen/lib/domgen/sql/model.rb:516:in `qualified_table_name'
/Users/Ian/Code/planner/vendor/plugins/domgen/lib/domgen/sync/templates/abstract_sync_temp_population_impl.java.erb:184